src/app/components/client/ProgressCard.tsx

```diff
@@ -67,31 +78,29 @@ export const ProgressCard = (props: Props) => {
   };
 
   const modalContent = (
-    <>
+    <div className={styles.modalBodyContent}>
       <p>
-        <strong>Resolved by you</strong> includes anything you have manually
-        fixed. All data breaches that require access to your accounts need to be
-        fixed manually, even if you have upgraded to Premium.
-        <br />
-        <br />
-        <strong>Auto-removed</strong> includes any exposures from data broker
-        profiles that we have removed for you. This is available only for
-        Premium subscribers. Complete includes anything resolved by you or
-        auto-removed by us.
-        <br />
-        <br />
-        <strong>In Progress</strong> includes anything that we are currently
-        working on fixing. Removals typically take 7-14 days but the most
-        difficult sites could take longer. You may also start to see removals
-        happening within the same day.
+        {l10n.getFragment("modal-heres-what-we-fixed-description-part-one", {
+          elems: { b: <strong /> },
```

Comment:
Ah, interesting!
Is this because the .ftl strings are using `<b>` and we need to remap to `<strong>` tags?

https://github.com/mozilla/blurts-server/blob/6da89960efaf57c44e00f45093a2055d8c16d681/pendingTranslations.ftl#L41-L51